scripts/nusrah-farri-ghazal.py

```diff
@@ -1 +1 @@
-print("Hello World, this is Nusrah Farri-Ghazal with HNGi7 ID HNG-00310 and email nusrahfarri@gmail.com using Python for stage 2 task")
+print("Hello World, this is Nusrah FarriGhazal with HNGi7 ID HNG-00310 using Python for stage 2 task")

```

Comment:
why did you remove the email from this file?

Reply:
sorry let me recommit